launcher/minecraft/mod/ModFolderModel.h

```diff
@@ -52,7 +73,7 @@ class ModFolderModel : public QAbstractListModel
         Enable,
         Toggle
     };
-    ModFolderModel(const QString &dir);
+    ModFolderModel(const QString &dir, bool is_indexed);
```

Comment:
```suggestion
    ModFolderModel(const QString &dir, bool is_indexed = false);
```

Seems slightly nicer to do this, and don't set it on the non-mod models, since only the mods set this to true for now